lua/rainbow-delimiters/strategy/global.lua

```diff
@@ -123,13 +130,17 @@ local function update_range(bufnr, changes, tree, lang)
 					matches:push(match_record)
 					match_record = nil
 				end
+			elseif starts_with_ignore(name) and match_record then
+				-- do nothing
+				goto continue
```

Comment:
`goto` is not part of Lua 5.1, it was added as an extension from a later version to LuaJIT. I know that *most* people will build Neovim with LuaJIT, but it is not guaranteed to be available on all platforms and in all distros. Do we really need a `goto` here? It looks to me like we would naturally get to the end of the if/else block.

```suggestion
```

Reply:
We don't need goto at all, but if it is just the comment, treesitter doesn't parse it nicely. I have changed it to an empty `do end` block now. I assume that should basically be equivalent to just a comment anyways.